src/main/java/gregtech/api/recipes/logic/ParallelLogic.java

```diff
@@ -440,9 +440,12 @@ public static RecipeBuilder<?> doParallelRecipes(Recipe currentRecipe, RecipeMap
         int limitByOutput;
         limitByOutput = ParallelLogic.limitByOutputMerging(currentRecipe, exportInventory, exportFluids, multiplierByInputs, voidItems, voidFluids);
 
-        int limitByVoltage = Math.abs((int) (maxVoltage / currentRecipe.getEUt()));
-        int parallelizable = Math.min(limitByVoltage, Math.min(multiplierByInputs, limitByOutput));
+        int parallelizable = Math.min(multiplierByInputs, limitByOutput);
```

Comment:
This implementation is bad practice, as if someone may need to rework it later he may miss the reassignment on line 447.
Also there is danger of returning incorrect value, in situation when `recipe.getEUt()` returns `0` as then parallelization value would be `Math.min(multiplierByInputs, limitByOutput)` which doesn't seems right.

Reply:
First of all, as it turns out, as `ParallelLogic.limitByOutputMerging` always returns a value less than or eqal to `multiplierByInputs`, I was able to remove that first assignment, hopefully in a way that is good practice.
I also tested this in my addon, and it worked to correctly run the right number of parallel recipes.